features/case-report.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const {Given, When, Then, setDefaultTimeout} = require('@cucumber/cucumber')
+
+const {
+  sendCovid19CaseReport,
+  verifyCovid19CaseReportInDhis,
+  verifyCovid19CaseReportInFhir,
+  cleanupCovid19CaseReport,
+  ensurePractitionerExists
+} = require('./utils')
+
+setDefaultTimeout(30000)
+
+Given('that the practitioner sending the reports exists', ensurePractitionerExists)
+
+When('a case report is sent through', sendCovid19CaseReport)
```

Comment:
The Questionnaires are not cleaned up post test - if this is simple could you please remove them

Reply:
This because we do the IG importation